docs/tutorial-golang.md

```diff
@@ -14,6 +14,14 @@ on how to setup the golang client using [Golang dep](https://github.com/golang/d
 To setup your environment, you will need to import the following to gain
 access to `api.pb.go`:
 
+#### Depedencies
+```
```

Comment:
This is good for simplicity. Move the recorded demo to this section and explain to use that for production.